components-comparison.html

```diff
@@ -0,0 +1,1484 @@
+                        </details>
+                    </td>
+                </tr>
+                <tr>
+                    <td class="c0">Built-in <abbr title="Components">Comp.</abbr></td>
+                    <td class="c1">span</td>
+                    <td class="c2"><code>&lt;span&gt;</code></td>
+                    <td class="c3"><code>&lt;span&gt;</code></td>
+                    <td class="c4 no"></td>
+                    <td class="c5 no"></td>
+                </tr>
+                <tr>
+                    <td class="c0">Built-in <abbr title="Components">Comp.</abbr></td>
+                    <td class="c1">stack</td>
+                    <td class="c2 no"></td>
+                    <td class="c3"><code>&lt;stack&gt;</code></td>
+                    <td class="c4">
+                        <details>
+                            <summary>
+                                <code>&lt;cover-view&gt;</code>
+                            </summary>
+                            <code>&lt;cover-view&gt;</code>, <code>&lt;cover-image&gt;</code>
+                        </details>
+                    </td>
+                    <td class="c5">
+                        <details>
+                            <summary>
+                                <code>&lt;cover-view&gt;</code>
+                            </summary>
+                            <code>&lt;cover-view&gt;</code>, <code>&lt;cover-image&gt;</code>
+                        </details>
+                    </td>
+                </tr>
+                <tr>
+                    <td class="c0">Built-in <abbr title="Components">Comp.</abbr></td>
+                    <td class="c1">carousel</td>
+                    <td class="c2 no"></td>
+                    <td class="c3"><code>&lt;swiper&gt;</code></td>
+                    <td class="c4"><code>&lt;swiper&gt;</code></td>
+                    <td class="c5">
+                        <details>
+                            <summary>
+                                <code>&lt;swiper&gt;</code>
+                            </summary>
+                            <code>&lt;swiper&gt;</code>, <code>&lt;swiper-item&gt;</code>
+                        </details>
+                    </td>
+
+                </tr>
+                <tr>
+                    <td class="c0">Built-in <abbr title="Components">Comp.</abbr></td>
+                    <td class="c1">tabs</td>
+                    <td class="c2 partial"><a href="https://open-ui.org/components/tabs.research">OpenUI research</a></td>
+                    <td class="c3">
+                        <details>
+                            <summary>
+                                <code>&lt;tabs&gt;</code>
+                            </summary>
+                            <code>&lt;tabs&gt;</code>, <code>&lt;tab-bar&gt;</code>, <code>&lt;tab-content&gt;</code>
+                        </details>
+                    </td>
+                    <td class="c4 no"></td>
```

Comment:
The UI component framework for Alipay created by Ant Group—Ant Design Mini—supports tabs, though: https://mini.ant.design/components/tabs

Reply:
Thanks for the pointer. Fixed: 055e12c06d909bd52f78a7957ffbf0737d558a9c